app/main/views/drafts.py

```diff
@@ -277,6 +279,40 @@ def list_draft_services():
     return list_result_response(RESOURCE_NAME, services), 200
 
 
+@main.route('/draft-services/framework/<string:framework_slug>', methods=['GET'])
```

Comment:
I feel a more RESTful approach would be to have this new resource attached to /frameworks/<string:framework_slug>/draft-services
But, even if true, it would not be very practical?

Reply:
Ideally we'd like a single paginated `/draft-services` endpoint, with multiple filter options such as `supplier_id`, `framework`, `status` etc. However this would introduce a breaking change to the current non-paginated endpoint, which is definitely to be avoided. 

However... if I added `supplier_id` as an optional filter to this endpoint, we could move towards deprecating the other endpoint?

Reply:
That may be a good idea. However, if the deprecation would take much effort (as you need to change invocations from the frontend apps) it should probably be a tech debt ticket (or at least a separate PR). That's a call of judgement.

Reply:
Commit added!